{"uuid": "bd89a8ab-56dd-4fdc-ae88-079a5b800d67", "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: The requested resource could not be found, or a request was received using an HTTP method that is not supported by the mapped resource\n\n", "trace": "  File \"D:\\IT\\Sourcecode\\IRIS-mini-app-automation\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"D:\\IT\\Sourcecode\\IRIS-mini-app-automation\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"D:\\IT\\Sourcecode\\IRIS-mini-app-automation\\.venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"D:\\IT\\Sourcecode\\IRIS-mini-app-automation\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\IT\\Sourcecode\\IRIS-mini-app-automation\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1195, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\IT\\Sourcecode\\IRIS-mini-app-automation\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 922, in call_fixture_func\n    fixture_result = next(generator)\n  File \"D:\\IT\\Sourcecode\\IRIS-mini-app-automation\\tests\\test_recipe_input.py\", line 7, in driver\n    driver = create_driver()\n  File \"D:\\IT\\Sourcecode\\IRIS-mini-app-automation\\utils\\driver_factory.py\", line 11, in create_driver\n    driver = webdriver.Remote(\"http://localhost:4723/wd/hub\", desired_caps)\n  File \"D:\\IT\\Sourcecode\\IRIS-mini-app-automation\\.venv\\Lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 257, in __init__\n    super().__init__(\n    ~~~~~~~~~~~~~~~~^\n        command_executor=command_executor,\n        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n        options=dst_options,\n        ^^^^^^^^^^^^^^^^^^^^\n    )\n    ^\n  File \"D:\\IT\\Sourcecode\\IRIS-mini-app-automation\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 261, in __init__\n    self.start_session(capabilities)\n    ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^\n  File \"D:\\IT\\Sourcecode\\IRIS-mini-app-automation\\.venv\\Lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 346, in start_session\n    response = self.execute(RemoteCommand.NEW_SESSION, w3c_caps)\n  File \"D:\\IT\\Sourcecode\\IRIS-mini-app-automation\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 454, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"D:\\IT\\Sourcecode\\IRIS-mini-app-automation\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1753511178921, "stop": 1753511181040}], "start": 1753511178921, "stop": 1753511181183}